TWLight/applications/management/commands/notify_applicants_tou_changes.py

```diff
@@ -20,20 +23,28 @@ def handle(self, *args, **options):
                 status__in=[Application.PENDING, Application.QUESTION],
                 partner__status__in=[Partner.AVAILABLE],
                 editor__isnull=False,
-                agreement_with_terms_of_use=False
+                editor__user__userprofile__terms_of_use=False,
             )
-                .exclude(editor__user__groups__name="restricted")
-                .order_by("status", "partner", "date_created")
+            .exclude(editor__user__groups__name="restricted")
+            .order_by("status", "partner", "date_created")
         )
 
-        # Loop through the apps and add a comment.
+        # Loop through the apps and add a comment if twl_team hasn't commented already.
         for app in pending_apps:
-            comment = Comment(
-                content_object=app,
-                site_id=settings.SITE_ID,
-                user=twl_team,
-                # Translators: This comment is added to pending applications when our terms of use change.
-                comment=_("Our terms of use have changed. "
-                          "Your applications will not be processed until you log in and agree to our updated terms.")
-            )
-            comment.save()
+            if (
+                Comment.objects.filter(
+                    object_pk=str(app.pk), site_id=settings.SITE_ID, user=twl_team
+                ).count()
+                == 0
+            ):
```

Comment:
We have cases [like this](https://wikipedialibrary.wmflabs.org/applications/evaluate/10573/) where the coordinator responsible already has left a comment. Is there a reason we don't want to check for coordinator comments?

Reply:
Because there are lots of reasons for coordinators to comment on applications besides our terms of use changing. We will be changing our terms again in the future, so this should be a generally working tool.

Reply:
I could add a check for the last commented posted date. eg. maybe we only comment if the app meets the existing criteria and has also not had a comment in n days?

Reply:
yes, that would be awesome! We give coordinators the applicant's ToU info, which they should consider before approving an app, so this leaves some room for coordinator discretion.

Reply:
Not really a discretion if we think about it, more of a trusting coordinators to take care of it.

Reply:
Okay, added in a check to see if there were comments in in the last 8 days. No comment will be left if TWL has ever commented OR if anyone has commented in the last 8 days. If I run this command tomorrow, it should not comment on any applications that have had comments since the command was run the first time.